oxo.py

```diff
@@ -1,33 +1,99 @@
 class OxoBoard:
-    def __init__(self):
-        """ The initialiser. Initialise any fields you need here. """
-        raise NotImplementedError("TODO: implement __init__")
+    """
+        Number representations:
+            0 -> Empty
+            1 -> O
+            2 -> X
+
+        board -> Playable positions
+        board_width, board_height -> Board size
+        strike_combinations -> Each position for strike
+    """
+    board = {}
+
+    board_width = 3
+    board_height = 3
+
+    strike_combinations = [
```

Comment:
This approach is prone to copy-paste errors, and would be difficult to extend to larger boards. Could this list be initialised programmatically?